completion.go

```diff
@@ -39,10 +40,28 @@ var (
 	}
 )
 
-func buildCompletionCommand(appName string) *Command {
+const completionDescription = `Output shell completion script for bash, zsh, fish, or Powershell.
+Source the output to enable completion.
+
+# .bashrc
+source <($COMMAND completion bash)
+
+# .zshrc
+source <($COMMAND completion zsh)
+
+# fish
+$COMMAND completion fish > ~/.config/fish/completions/$COMMAND.fish
+
+# Powershell
+Output the script to path/to/autocomplete/$COMMAND.ps1 an run it.
+`
+
+func BuildCompletionCommand(appName string) *Command {
```

Comment:
Why the name change to make it public ?

Reply:
- https://github.com/urfave/cli/discussions/2102

To allow to change the completion command's filelds such as `Hidden`.